lib/impress.application.js

```diff
@@ -278,13 +278,17 @@ impress.application.mixin = (application) => {
     api.fs.readdir(absPath, (err, files) => {
       if (err) {
         application.log.error(impress.CANT_READ_DIR + absPath);
-        return callback(err);
+        return callback();
```

Comment:
I'm not sure about this change, though it is the matter of https://github.com/metarhia/Impress/issues/680 anyway.